libs/wingc/src/jsify.rs

```diff
@@ -1066,23 +1077,31 @@ impl<'a> JSifier<'a> {
 			.iter()
 			.map(|inner_member_name| format!("{}: ${{{}_client}}", inner_member_name, inner_member_name))
 			.join(", ");
-		formatdoc!("
+		formatdoc!(
+			"
 			_toInflight() {{
 				{inner_clients}
 				const self_client_path = {client_path};
-				return {STDLIB}.core.NodeJsCode.fromInline(`(new (require(\"${{self_client_path}}\")).{resource_name}({{{captured_fields}}}))`);
+				return {STDLIB}.core.NodeJsCode.fromInline(`(
+					await (async () => {{ 
```

Comment:
Interesting... This implies that `_toInflight()` is called from an async scope. Is that the case? 

Feels like this is an assumption we should model somehow into the API. Otherwise, someone will assume in the future that it's a sync call.

@Chriscbr curious what you think...

Reply:
`_toInflight()` is still sync -- do you mean to say the code fragment returned by `toInflight()` is called from an async scope? Yes, I agree it feels like should that should be documented it in the `_toInflight()` API since it means you can't call initialize one of the clients from an ordinary JavaScript constructor for example.

Technically I guess this isn't a big change since that is already how we are invoking some inflight code right now (see below), but something to keep our eyes on if we ever need to initialize inflight code from plain javascript constructors for some reason.

```js
// this is the structure of the code that the SDK bundles into an AWS Lambda function
exports.handler = async function(event) {
  return await (< inflight._toInflight().text >).handle(event);
};
```